app/models/form.rb

```diff
@@ -34,6 +34,10 @@ def is_archived?
     state.to_sym.in?(%i[archived archived_with_draft])
   end
 
+  def is_archive_with_draft?
+    state.to_sym == :archived_with_draft
+  end
```

Comment:
The commit message adding this method says it will be needed in the task list, but I can't see it used anywhere 🤔

Reply:
ah crumbs I thought I removed it as part of dropping a commit - rebase error